_tools/run/helpers/paths/htmlFilePaths.js

```diff
@@ -0,0 +1,90 @@
+// Lint with JS Standard
+
+// Import Node modules
+const fsPath = require('path')
+const fileList = require('./fileList.js')
+const works = require('./works')
```

Comment:
Nitpicky, but do we want consistency of use of file extension in require statements? E.g. './fileList.js' vs './works'

Reply:
Thanks, consistency would be great. Fixed in [ebc6b9d](https://github.com/electricbookworks/electric-book/pull/731/commits/ebc6b9d55115051ee4c758936301a12ef35310d3).